### PR TITLE
Readme

### PR DESCRIPTION
**Pull request by abhayks1**
PR Contains below:
- Readme update.
- It contains deployment of contracts.
- It contains direct transfer of tokens.
- it contains payment through PricerRule.
- It contains Wallet operations.

Note
- More description might be needed for some section.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,596 +1,493 @@
...
 
-// organization address
-let organizationAddress = devEnvConfig.organizationAddress;
+* [TokenHolder](https://github.com/OpenSTFoundation/openst-contracts/blob/0.10.0-alpha.1/contracts/token/TokenHolder.sol) (TokenHolder contract deployed on UtilityChain)
+* [TokenRules](https://github.com/OpenSTFoundation/openst-contracts/blob/0.10.0-alpha.1/contracts/token/TokenRules.sol) (TokenRules contract deployed on UtilityChain)
+* [PricerRule](https://github.com/OpenSTFoundation/openst-contracts/blob/0.10.0-alpha.1/contracts/rules/PricerRule.sol) (PricerRule is a rule contract deployed on UtilityChain)
+* [GnosisSafe](https://github.com/gnosis/safe-contracts/blob/v0.1.0/contracts/GnosisSafe.sol) (MultiSignature wallet with support for confirmations using signed messages)
+* [DelayedRecoveryModule](https://github.com/OpenSTFoundation/openst-contracts/blob/0.10.0-alpha.1/contracts/gnosis_safe_modules/DelayedRecoveryModule.sol) (Allows to replace an owner without Safe confirmations) 
+* [CreateAndAddModules](https://github.com/gnosis/safe-contracts/blob/v0.1.0/contracts/libraries/CreateAndAddModules.sol) (Allows to create and add multiple module in one transaction)
```
Comment (benjaminbollen) [inline README.md]: `CreateAndAddModules` this probably does not need to be highlighted to the end-user

Comment (benjaminbollen) [reply README.md]: can remain here in the readme for now; but the user reading will definitely get confused.  Perhaps the `## ABI and BIN provider` can move down in the README

File: README.md [hunk 2]
```diff
@@ -1,596 +1,493 @@
...
+An Organization contract serves as an on-chain access control mechanism by assigning roles to a set of addresses.
 
 ```js
-// Here is a sample helper method for developers to add an account to the wallet using keystore content
-// Read more about web3.eth.accounts.decrypt here: https://web3js.readthedocs.io/en/1.0/web3-eth-accounts.html#id22
-let addToWalletByKeystoreContent = function (encryptedPrivateKey, password) {
-  let account = web3.eth.accounts.decrypt(encryptedPrivateKey, password);
-  web3.eth.accounts.wallet.add(account);
+// Deploy Organization contract
+const Mosaic = require('@openstfoundation/mosaic.js');
```
Comment (benjaminbollen) [inline README.md]: we should highlight on top of the readme that for organization you need this dependency

File: README.md [hunk 3]
```diff
@@ -1,596 +1,493 @@
...
-
-```bash
-node ./node_modules/\@openstfoundation/openst.js/tools/initDevEnv.js ~
-
-
-```
-
-#### Creating an OpenST object
-OpenST.js is a thin layer over web3.js. Its constructor arguments are same as that of web3.js.
+This code was tested with geth version: 1.7.3-stable. Other higher versions should also work.
```
Comment (benjaminbollen) [inline README.md]: this needs to be updated; we re testing with geth 1.8.23

Comment (benjaminbollen) [reply README.md]: granted, we re not using any of the new EIPs; perhaps just remove this line; state that it works for Ethereum Byzantium and Petersburg

File: README.md [hunk 4]
```diff
@@ -1,596 +1,493 @@
...
 ============
 
-OpenST is a framework for building token economies. Here are steps to get started with OpenST.js. In order to view an example, please visit our example repo, [OpenST.js Examples](https://github.com/OpenSTFoundation/openst-js-examples).
+![Build master](https://img.shields.io/travis/OpenSTFoundation/openst.js/master.svg?label=build%20master&style=flat)
+![Build develop](https://img.shields.io/travis/OpenSTFoundation/openst.js/develop.svg?label=build%20develop&style=flat)
+![npm version](https://img.shields.io/npm/v/@openstfoundation/openst.js.svg?style=flat)
+![Discuss on Discourse](https://img.shields.io/discourse/https/discuss.openst.org/topics.svg?style=flat)
+![Chat on Gitter](https://img.shields.io/gitter/room/OpenSTFoundation/SimpleToken.svg?style=flat)
 
-#### Overview of different components:
```
Comment (benjaminbollen) [inline README.md]: this high level description (with minor update to the TokenHolder) is quite valuable to have at the top of the readme

File: README.md [hunk 5]
```diff
@@ -1,596 +1,493 @@
 OpenST.js
 ============
 
-OpenST is a framework for building token economies. Here are steps to get started with OpenST.js. In order to view an example, please visit our example repo, [OpenST.js Examples](https://github.com/OpenSTFoundation/openst-js-examples).
```
Comment (benjaminbollen) [inline README.md]: also let s keep this short descriptiion of OpenST

File: README.md (hunk 2 above)
Comment (abhayks1) [reply README.md]: Updated in "Creating an OpenST object" section.

File: README.md (hunk 4 above)
Comment (abhayks1) [reply README.md]: I think this should be part of openst-contracts as that seems to be ideal placeholder 🤔 . 
We can point to openst-contracts readme through github url.

**Comment by abhayks1** [comment]
@benjaminbollen I haven't tested the readme steps.
Should I test the readme steps once or we should do that with JLP 🤔 ?

**Comment by benjaminbollen** [comment]
let's test them as we build JLP's use of OpenST.js alpha.4

**Pull request merged**